storage/mounts_test.go

```diff
@@ -86,6 +87,7 @@ func TestAccSourceOnInvalidMountFails(t *testing.T) {
 
 func TestAccInvalidSecretScopeFails(t *testing.T) {
```

Comment:
and `TestAcc` prefix is for acceptance tests, so this means that `common.CommonEnvironmentClient()` could be used :)